_includes/head.html

```diff
@@ -5,6 +5,13 @@
 
     <title>Indico - {% if page.title %}{{ page.title }}{% else %}{{ site.title }}{% endif %}</title>
     <meta name="description" content="{{ site.description }}">
+    <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
+
+    <meta property="og:url" content="{{ page.url | replace:'index.html','' | prepend: site.baseurl | prepend: site.url }}">
```

Comment:
i'd either put a space after every `:` or after none - but not a mix of both (IMHO the no-space version is easier to read)

(also WTF is this weird template engine with such an obnoxious filter syntax?!)

Reply:
This was a 1:1 copy of what we are doing in the canonical section. https://github.com/indico/indico.github.io/blob/6e766680c68b7ebe3fb573539f6fda3b44a3f310/_includes/head.html#L15

But sure, I'll remove the excess spaces.